docs/rollback-reaction.md

```diff
@@ -0,0 +1,98 @@
+# Reaction on rollback
+
+This document describes how one should react to rollbacks.
+
+## Overview
+
+Rollback happens when node's blockchain `A` is not a prefix of another
+valid blockchain `B` which has more blocks than `A`. In this case node
+will rollback all blocks from `A` which are not in `B` and apply all
+blocks from `B` which are not in `A`.  Rollback itself doesn't mean
+that the system is broken. It's an integral part of many, if not all,
+cryptocurrencies. However, if all stakeholders are honest, always
+online, don't have bugs and there are no delays (e. g. networking),
+rollbacks are impossible.  In real world only the first condition is
+satisfied in Byron (all stakeholders are indeed honest). And it
+significantly decreases chances of rollback, but not fully.
+
+Examples:
+1. Node `N` creates a block `B` and is immediately stopped before it
+   announces it to other nodes. After the node is started again, other
+   nodes will create more blocks and `N` will have to rollback `B`.
+2. Another example: there is a huge delay in blocks propagation
+   (unstable network) and because of that some node may not receive
+   block for slot `i` before it ends. And then create block for slot
+   `i + 1` (not based on `i`). It will also lead to rollback.
+
+## How to notice rollback
+
+There are several ways to notice that rollback occurred:
+1. Messages like these:
+  * > Blocks to rollback [62955975]
```

Comment:
I couldn't understand it's a block hash inside square brackets for 20s lol. Can you add some a-f characters maybe? :)